targets/TARGET_TOSHIBA/TARGET_TMPM46B/device/TOOLCHAIN_ARM_STD/tmpm46bf10fg.sct

```diff
@@ -31,6 +39,11 @@
 #endif
 
 #define Stack_Size MBED_CONF_TARGET_BOOT_STACK_SIZE
+#define VECTOR_SIZE 0x320
```

Comment:
Should this be `0x1E0`?

Reply:
Thanks! This was a remnant from when I copied changes from the other targets scatter files.